### ISSUE TITLE
German Translation for "End meeting for all" not working

### ISSUE THREAD

**Issue by Daniel-at-git** [bigbluebutton/bigbluebutton#19794]
**Description**
Since Upgrade to 2.7.5 the German translation shows "End meeting for all" in the menu.
![bbb-translation](https://github.com/bigbluebutton/bigbluebutton/assets/63794766/5e98fe2d-97ae-4474-b62a-d17afcd6dc03)

Tested on [demo](https://demo.bigbluebutton.org)
Maybe it's related to https://github.com/bigbluebutton/bigbluebutton/pull/19232

**To Reproduce**
1. Connect to room
1. Switch to German
2. Go to the menu (right upper corner)
3. Take a look to the last line

**Expected behavior**
Line should be "Konferenz für alle beenden" as configured in
https://github.com/bigbluebutton/bigbluebutton/blob/v2.7.x-release/bigbluebutton-html5/public/locales/de.json#L460

**Actual behavior**
"End meeting for all" is displayed.

**BBB version:**
$ sudo bbb-conf --check | grep BigBlueButton
BigBlueButton Server 2.7.5 (579)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Chrome
 - Version: 122.0.6261.128

**Comment by ffdixon** [comment]
Good catch!  It looks like the localization string is there in transifex.  

<img width="1044" alt="image" src="https://github.com/bigbluebutton/bigbluebutton/assets/290351/99507d53-fd8e-4032-96ea-eb1c16c68111">

We'll check that it propagates to the client in the next update.

**Comment by antobinary** [comment]
Hi! This was handled via https://github.com/bigbluebutton/bigbluebutton/pull/19768 and https://github.com/bigbluebutton/bigbluebutton/pull/19771
Expect the translation as part of the next 2.7.x release likely later this month. Alternatively, pull `de.json` from https://github.com/bigbluebutton/bigbluebutton/blob/v2.7.x-release/bigbluebutton-html5/public/locales/de.json and replace your local de.json with it.

**Comment by Daniel-at-git** [comment]
Thanks @antobinary for the hint. I replaced
/usr/share/meteor/bundle/programs/web.browser.legacy/app/locales/de.json
/usr/share/meteor/bundle/programs/web.browser/app/locales/de.json
(and restarted bbb) but it  still shows englisch text.
```bash
# md5sum /usr/share/meteor/bundle/programs/web.browser.legacy/app/locales/de.json /usr/share/meteor/bundle/programs/web.browser/app/locales/de.json
396af16f1794f03fa0a0d39f80e1dab9  /usr/share/meteor/bundle/programs/web.browser.legacy/app/locales/de.json
396af16f1794f03fa0a0d39f80e1dab9  /usr/share/meteor/bundle/programs/web.browser/app/locales/de.json
```

**Comment by antobinary** [comment]
I just did these steps on test27.bigbluebutton.org
I replaced
`/usr/share/meteor/bundle/programs/web.browser.legacy/app/locales/de.json` and
`/usr/share/meteor/bundle/programs/web.browser/app/locales/de.json` with
https://raw.githubusercontent.com/bigbluebutton/bigbluebutton/v2.7.x-release/bigbluebutton-html5/public/locales/de.json

restated BBB
Created a meeting (my locale was already German as I tested before). I could see the English "End meeting for all".
*I cleared browser cache* and now I see it correctly in German.

**Comment by antobinary** [comment]
Please let me know if it worked correctly for you too. I am thinking of documenting this step.

**Comment by Daniel-at-git** [comment]
@antobinary cleaning the browser cache solved it :-D

**Comment by antobinary** [comment]
https://github.com/bigbluebutton/bigbluebutton/pull/19798

**Comment by Daniel-at-git** [comment]
After one day everything is englisch now and didn't get it back. Also not in a private tab :-o
I went back to original 2.7.6 de.json. I need to clean the cache in the browser. Not a good option for all other normal users....

**Comment by antobinary** [comment]
Hi @Daniel-at-git 
Could you elaborate on what happened? I see https://groups.google.com/g/bigbluebutton-dev/c/Jx5kpDKb1Lg/m/tFjB8sYLAwAJ
If you're not able to patch this, let me know, I may be able to release 2.7.6 ~tomorrow with the correct localizations.

**Comment by Daniel-at-git** [comment]
@antobinary I patched it with your file from
https://raw.githubusercontent.com/bigbluebutton/bigbluebutton/v2.7.x-release/bigbluebutton-html5/public/locales/de.json
and replaced both files
/usr/share/meteor/bundle/programs/web.browser.legacy/app/locales/de.json
/usr/share/meteor/bundle/programs/web.browser/app/locales/de.json

AFAIK it has been (after restart and clear browser cache) working fine. One day later everything(!) is english.

So I replaced de.json with the original version from 2.7.5. It's working again if I clear my browser cache. My problem is: I can not tell all user to clear the browser cache. So I'm looking for a solution to force this from server side. AFAIK it would help if I modify the etag which is send in every HTTP answer.

A 2.7.6 can help for new etags also and all other German people would be happy too. So a new release would be great :-D

**Comment by antobinary** [comment]
@Daniel-at-git 2.7.6 is out!
